changes/21256-fleet-desktop-trap

```diff
@@ -0,0 +1 @@
+- Gracefully shutdown fleet desktop when receiving interrupt and terminate signals
```

Comment:
Please move to `orbit/changes/`.

Reply:
Double checking in case you missed to push (marked as resolved).

Reply:
I forgot to push, done now 🙂